core/js/home.script.js

```diff
@@ -75,3 +81,34 @@ function updateCounter(new_value, classname)
 		}
 	});
 })();
+
+// Array with timer IDs
+var timers = [];
+
+function startTimer(duration, element)
+{
+	var countdown = duration, hours, minutes, seconds;
+	timers.push(setInterval(function () {
+		hours = Math.abs(parseInt(countdown / 3600, 10));
+		minutes = Math.abs(parseInt((countdown / 60) % 60, 10));
+		seconds = Math.abs(parseInt(countdown % 60, 10));
+
+		hours = hours < 10 ? "0" + hours : hours;
+		minutes = minutes < 10 ? "0" + minutes : minutes;
+		seconds = seconds < 10 ? "0" + seconds: seconds;
+
+		var output = hours + ":" + minutes + ":" + seconds
+		if (--countdown >= 0) {
+			$(element).text(output);
+		} else {
+			$(element).text("- " + output);
+			$(element).css({ 'color': 'rgb(200, 50, 50)'});
+		}
+	}, 1000));
+}
+
+function stopTimer()
```

Comment:
clears the interval (stops the timer) of the last element on the page + removes this element from the timer array